src/main.py

```diff
@@ -1,4 +1,44 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import sys
+
 from seedsigner.controller import Controller
 
-# Get the one and only Controller instance and start our main loop
-Controller.get_instance().start()
+logger = logging.getLogger(__name__)
+
+
+def main(sys_argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-l",
+        "--loglevel",
+        choices=list((logging._nameToLevel.keys())),
+        default="INFO",
+        type=str,
+        help=(
+            "Set the log level (default: %(default)s), WARNING: changing the log level "
+            "to something more verbose than %(default)s may result in unwanted data "
+            "being written to stderr"
+        ),
+    )
+
+    args = parser.parse_args(sys_argv)
+
+    root_logger = logging.getLogger()
+    root_logger.setLevel(logging.getLevelName(args.loglevel))
+    console_handler = logging.StreamHandler(sys.stderr)
+    console_handler.setFormatter(
+        logging.Formatter("%(asctime)s.%(msecs)03d %(levelname)s:\t%(message)s")
```

Comment:
uh. thanks for the redundant msec finding, don't know why i didn't see that... this seems to have changed, I remember that is was required when working with older Python versions... having msecs help to accurately see how much time is required between certain calls.

Good idea to add funcname + lineno etc to the log line... (I am used to have all those fields via json loggers, which are not feasible here for the console) ... went for `"%(asctime)s %(levelname)8s [%(name)s %(funcName)s (%(lineno)d)]: %(message)s"` (=changed the 5s to 8s spacing so that all the potential loglevel are right aligned .. longest name is `CRITICAL`), see https://github.com/SeedSigner/seedsigner/pull/558/commits/5a4ae50cf0210d489bc7d0e2ac5b447c577a3b48